moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -121,13 +121,36 @@ trajectory_msgs::msg::JointTrajectory composeTrajectoryMessage(const servo::Para
 
   trajectory_msgs::msg::JointTrajectoryPoint point;
```

Comment:
You could make this static so it's not re-created every time.

```suggestion
  static trajectory_msgs::msg::JointTrajectoryPoint point;
```